src/api/liskApi.js

```diff
@@ -12,43 +12,11 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-
-/**
- * LiskAPI module provides functions for interfacing with the Lisk network.
- * Providing mechanisms for:
- *
- * - Retrieval of blockchain data: accounts, blocks, transactions.
- * - Enhancing Lisk security by local signing of transactions and immediate network transmission.
- * - Connecting to Lisk nodes or to localhost instance of Lisk core.
- * - Configurable network settings to work in different Lisk environments.
- *
- *     var lisk = require('lisk-js');
- *
- *     var options = {
- *         ssl: true,
- *         node: '',
- *         randomNode: true,
- *         testnet: true,
- *         port: '7000',
- *         bannedNodes: [],
- *         nodes: [],
- *         nethash: ''
- *     };
- *
- *
- *     var LSK = lisk.api(options);
-*/
```

Comment:
This isn't actually a yuidoc comment, but I'm happy with it being removed as we will have documentation on the documentation site. What do you think @Tosch110 ?

Reply:
Yes, agree! It is actually there for yuidoc, it would show it as an example also in yuidoc.